barretenberg/cpp/scripts/cmake/preset-build-dir

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
this is kinda crazy, hope we can move away from it in the future with a more standardized build folder scheme (with a symlink called build for whatever the native preset is)